lib/main.dart

```diff
@@ -49,57 +51,57 @@ class _PerkPageState extends State<PerkPage> {
     });
   }
 
+  Color themeSelection() {
+    var colorSelected;
```

Comment:
These 3 lines can be simplified into 1:
`return lightModeDisabled ? Color(0xff21213b) : Colors.blueGrey.shade100`